modules/apps/social/social-activity-impl/src/main/java/com/liferay/social/activity/internal/manager/SocialActivityManagerImpl.java

```diff
@@ -94,10 +94,6 @@ public void addUniqueActivity(
 
 	@Override
 	public void deleteActivities(T model) throws PortalException {
-		if (!_isEnabled(model.getCompanyId())) {
-			return;
-		}
-
```

Comment:
Have you tested what happens when `deleteActivities` is triggered for a company having the property `enableUserSocialActivityTracking` disabled? The lack of activities is just ignored or an exception is thrown?

Reply:
Hi Marco, when enableUserSocialActivityTracking is disabled, there are no errors. During the deletion, no records are found and there is simply nothing to delete.